src/code/audio/code_8019AF00.c

```diff
@@ -1,6 +1,1880 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/D_801E0BD0.s")
+typedef struct {
+    /* 0x0 */ s8 x;
+    /* 0x1 */ s8 y;
+} OcarinaControlStick; // size = 0x2
+
+typedef struct {
+    /* 0x0 */ u16 playerIO;
+    /* 0x2 */ u16 channelMask;
+    /* 0x4 */ u8 channelIO[3 * 33 + 1];
+} NatureAmbienceDataIO; // size = 0x68
+
+typedef struct {
+    /* 0x0 */ Vec3f* pos;
+    /* 0x4 */ f32 freqScale;
+    /* 0x8 */ s8 reverbAdd;
+} SfxSettings; // size = 0x9
+
+typedef struct {
+    /* 0x0 */ f32 vol;
+    /* 0x4 */ f32 freqScale;
+    /* 0x8 */ s8 reverb;
+    /* 0x9 */ s8 panSigned;
+    /* 0xA */ s8 stereoBits;
+    /* 0xB */ u8 filter;
+    /* 0xC */ u8 unk_0C;
+    /* 0xD */ u8 unk_0D;
+} SfxPlayerState; // size = 0xE
+
+typedef enum {
+    /* 0x0 */ SFX_CHANNEL_PLAYER0, // SfxPlayerBank
+    /* 0x1 */ SFX_CHANNEL_PLAYER1,
+    /* 0x2 */ SFX_CHANNEL_PLAYER2,
+    /* 0x3 */ SFX_CHANNEL_ITEM0, // SfxItemBank
+    /* 0x4 */ SFX_CHANNEL_ITEM1,
+    /* 0x5 */ SFX_CHANNEL_ENV0, // SfxEnvironmentBank
+    /* 0x6 */ SFX_CHANNEL_ENV1,
+    /* 0x7 */ SFX_CHANNEL_ENV2,
+    /* 0x8 */ SFX_CHANNEL_ENEMY0, // SfxEnemyBank
+    /* 0x9 */ SFX_CHANNEL_ENEMY1,
+    /* 0xA */ SFX_CHANNEL_ENEMY2,
+    /* 0xB */ SFX_CHANNEL_SYSTEM0, // SfxSystemBank
+    /* 0xC */ SFX_CHANNEL_SYSTEM1,
+    /* 0xD */ SFX_CHANNEL_OCARINA, // SfxOcarinaBank
+    /* 0xE */ SFX_CHANNEL_VOICE0,  // SfxVoiceBank
+    /* 0xF */ SFX_CHANNEL_VOICE1
+} SfxChannelIdx; // playerIdx = 2
+
+typedef enum {
+    /* 0x0 */ PLAYER_IO_PORT_0,
+    /* 0x1 */ PLAYER_IO_PORT_1,
+    /* 0x2 */ PLAYER_IO_PORT_2,
+    /* 0x3 */ PLAYER_IO_PORT_3,
+    /* 0x4 */ PLAYER_IO_PORT_4,
+    /* 0x5 */ PLAYER_IO_PORT_5,
+    /* 0x6 */ PLAYER_IO_PORT_6,
+    /* 0x7 */ PLAYER_IO_PORT_7,
+} PlayerIOPort;
```

Comment:
What is PLAYER in this context?
Usually we use PLAYER to refer to the user who is playing the game (typically controlling Link), but this wouldn't make sense here.
Maybe a controller attached to the console? But the N64 can connect up to 4 controllers, not 8.

So, what is this? I think this should have a comment describing what this enum is

Reply:
Yes, this is something I've brought up with OoT before as the term `PLAYER` is indeed overloaded and is quite unfortunate.

https://discord.com/channels/688807550715560050/688851317593997489/892952563434012693

There is Link Player and there are Audio Players, which house the different songs/sfx's that can be played. MM has 5 players that are represented by this enum:
https://github.com/engineer124/mm/blob/7ea60a08c675c00140a038df40276216ada2ba9c/include/sequence.h#L136-L142

If there is a better alternative to `Player` for this, I would support it. But I haven't been able to think of one. And this would need to change for OoT as well.

I suppose for now, I'll switch this enum to `SEQ_PLAYER` for consistency (which this enum and for the file `audio_seqplayer`